Simpson_Assign1/Program.cs

```diff
@@ -16,11 +16,131 @@ static void Main(string[] args)
+                        }
+                        Console.WriteLine("");
+                        break;
+                    case ("6"):
+                        Console.Write("Please enter the Z-ID <omitting the Z character> of the student you like to enroll into a course.");
+                        string zid = Console.ReadLine();
+                        Console.WriteLine("Which course will this student be enrolled into?");
+                        Console.Write("<DEPT COURSE_NUM-SECTION_NUM> ");
+                        string enrollcrs = Console.ReadLine();
+                        string[] words3 = enrollcrs.Split(' ');
+                        string[] words4 = words3[1].Split('-');
+                        List<Course> foundCourse2 = Courses.FindAll(x => x.DepartmentCode == words3[0] && x.CourseNumber == Convert.ToUInt64(words4[0]) && x.SectionNumber == words4[1]);
+                        List<Student> foundStudent4 = Students.FindAll(x => x.ZId == Convert.ToUInt64(zid));
+                        int success = foundStudent4[0].Enroll(foundCourse2[0]);
+                        if(success == 0)
+                        {
+                            Console.WriteLine("\nz{0} has successfully enrolled into {1}.", zid, enrollcrs);
+                        }
+                        Console.WriteLine("");
+                        break;
+                    case ("7"):
+                        Console.Write("Please enter the Z-ID <omitting the Z character> of the student you like to drop from a course.");
+                        string zid2 = Console.ReadLine();
+                        Console.WriteLine("Which course will this student be dropped from?");
+                        Console.Write("<DEPT COURSE_NUM-SECTION_NUM> ");
+                        string dropcrs = Console.ReadLine();
+                        string[] words5 = dropcrs.Split(' ');
+                        string[] words6 = words5[1].Split('-');
+                        List<Course> foundCourse3 = Courses.FindAll(x => x.DepartmentCode == words5[0] && x.CourseNumber == Convert.ToUInt64(words6[0]) && x.SectionNumber == words6[1]);
+                        List<Student> foundStudent5 = Students.FindAll(x => x.ZId == Convert.ToUInt64(zid2));
+                        int success2 = foundStudent5[0].Drop(foundCourse3[0]);
+                        if (success2 == 0)
+                        {
+                            Console.WriteLine("\nz{0} has successfully dropped from {1}.", zid2, dropcrs);
+                        }
+                        Console.WriteLine("");
+                        break;
+                    default:
+                        break;
+                }
             }
-            while (Console.ReadLine() != "8");
+            while (input != "8" && input != "h" && input != "q" && input != "Q" && input != "quit" && input != "Quit" && input != "exit" && input != "Exit");
```

Comment:
convert everything to capitalized before doing the comparison.

For Example:
input.ToUpper() != "H"
input.ToUpper() != "EXIT"

This will make the conditional statement shorter because you don't have to explicitly check both "exit" AND "Exit" and it will also work if the user tries to be cute and does something like "eXiT"